mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/options/AuthenticationMechanismType.java

```diff
@@ -39,7 +44,25 @@ public MongoCredential createCredential(String username, String databaseName, St
 		public MongoCredential createCredential(String username, String databaseName, String password) {
 			return MongoCredential.createMongoX509Credential( username );
 		}
-	};
+	}
+	,
+	SCRAM_SHA_1 {
```

Comment:
Could you update the docs of `MongoDBProperties.AUTHENTICATION_MECHANISM` to state the new default value? I don't think we need to list the actual values there, a ref to the enum is good enough.